### DISCUSSION TITLE
I receive notifications from many other users about fork projects.

### DISCUSSION THREAD

**Discussion by 521xueweihan** [community/community#39938] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If someone forks my project and posts a comment on a commit of the forked project.

Should I have received these notices?

Look at what I've been through：
<img width="1168" alt="截屏2022-11-24 21 06 00" src="https://user-images.githubusercontent.com/8255800/203792097-6cf685f6-cb1b-4d7b-b0a7-fbe4ee598185.png">

Such as url：https://github.com/kxg199/HelloGitHub/commit/33d9d83add15c6ff69b381eeb355cc4a7d8f16d5#commitcomment-90955238

**Comment by ossanna16** [comment]
Hi there @521xueweihan and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
